tests/integration/test_chat_async.py

```diff
@@ -149,7 +150,13 @@ async def test_serializer_hook():
 
     resp = await communicator.receive_json_from()
     resp.pop('time')
-    assert resp == {'room': room.pk, 'text': 'hello', 'type': 'chat', 'user': user.pk}
+    assert resp == {
+        'room': room.pk,
+        'text': 'hello',
+        'type': 'chat',
+        'user': user.pk,
+        'id': (await get_messages_from_room(room))[-1].id
```

Comment:
what if this fails, or will it never fail?

Reply:
If this fails, we should know about it. It means there was no message object created, which is a problem.

Reply:
It should never fail if things are working as expected.

Reply:
ugh my bad, didn't notice this was in a test ... ! ty